wiki/Mascots/de.md

```diff
@@ -92,7 +92,7 @@ Chirou wurde von [pyun](https://osu.ppy.sh/users/981534) entworfen und war eine
 
 ![Taikonator](img/Taikonator.png "Taikonator")
 
-Taikonator, auch bekannt als Taikonator 3000, war einer der Teilnehmenden am Designwettbewerb für das Maskottchen von osu!taiko. Er hat aus unbekannten Gründen als Insider-Witz eine gewisse Zugkraft und Popularität erlangt. Seine Herkunft bleibt ein Rätsel; trotzdem ist er weitaus einzigartiger als die anderen Designvorschläge für das osu!taiko-Maskottchen und hat viele Auftritte in verschiedenen Fanart-Beiträgen von Benutzern.
+Taikonator wurde von [Lemia-Chan](https://osu.ppy.sh/users/8506749), auch bekannt als Taikonator 3000, war einer der Teilnehmenden am Designwettbewerb für das Maskottchen von osu!taiko. Er hat aus unbekannten Gründen als Insider-Witz eine gewisse Zugkraft und Popularität erlangt. Seine Herkunft bleibt ein Rätsel; trotzdem ist er weitaus einzigartiger als die anderen Designvorschläge für das osu!taiko-Maskottchen und hat viele Auftritte in verschiedenen Fanart-Beiträgen von Benutzern.
```

Comment:
This sentence makes no sense at all now. 

```suggestion
Taikonator, auch bekannt als Taikonator 3000, war einer der Teilnehmenden am Designwettbewerb für das Maskottchen von osu!taiko und wurde von [Lemia-Chan](https://osu.ppy.sh/users/8506749) designt. Taikonator hat aus unbekannten Gründen als Insider-Witz eine gewisse Zugkraft und Popularität erlangt. Seine Herkunft bleibt ein Rätsel; trotzdem ist er weitaus einzigartiger als die anderen Designvorschläge für das osu!taiko-Maskottchen und hat viele Auftritte in verschiedenen Fanart-Beiträgen von Benutzern.
```

Reply:
the wording is copied from the other instances of this in the same article (though perhaps i did it incorrectly), should i update those too?

Reply:
The insertation for Taikonator may make it seem like it's refering to Lemia-Chan, basically saying that they are known for being called Taikonator 3000. If you take a second and focus on reading the sentence, you can understand what it's actually trying to say but the current structure is not very readable.

Reply:
yea the verb stating what she actually did (desgining) is missing and the entire sentence kinda gets messed up with the insertion as now everything references her rather than taikonator.